packages/get-platform/src/getPlatform.ts

```diff
@@ -132,20 +133,32 @@ export function parseOpenSSLVersion(input: string): string | undefined {
 }
 
 // getOpenSSLVersion returns the OpenSSL version excluding the patch version, e.g. "1.1.x"
-export async function getOpenSSLVersion(): Promise<string | undefined> {
+export async function getOpenSSLVersion(
+  platform: NodeJS.Platform,
+  arch: Arch,
+): Promise<string | undefined> {
+  let prefix: string | undefined = undefined
+
+  if (platform == 'android') {
+    prefix = '/data/data/com.termux/files/usr'
+  }
+
+  const isArmAndAndroid =
+    (arch === 'arm' || arch === 'arm64') && platform == 'android'
   const [version, ls] = await Promise.all([
     gracefulExec(`openssl version -v`),
     gracefulExec(`
-      ls -l /lib64 | grep ssl;
-      ls -l /usr/lib64 | grep ssl;
+      ls -l ${prefix ? prefix : ''}/lib${
+      isArmAndAndroid ? '' : '64'
+    } | grep ssl;
+      ${platform != 'android' && `ls -l /usr/lib | grep ssl;`}
```

Comment:
```suggestion
      ${platform != 'android' && `ls -l /usr/lib64 | grep ssl;`}
```

to at least make it work like before the changes on non-Android Linux, but better yet, see my previous comment for a more general and maintainable approach.

Reply:
Quoting that comment again,

> The best approach here would be to have an array of the library paths that depends on the platform and architecture (and optionally distro, but mixing `debian`-specific and `redhat`-specific paths together is also fine), and then doing something like ``libraryPaths.map(path => `ls -l ${path} | grep ssl;`).join('\n')`` here.
>
> So you'd have all of these checks separately, and an array like `['/data/data/com.termux/files/usr/lib']` for Android/Termux, `['/lib64', '/usr/lib64', '/lib/x86_64-linux-gnu', '/usr/lib/x86_64-linux-gnu', '/lib', '/usr/lib']` for Linux on x86, `['/lib64', '/usr/lib64', '/lib/aarch64-linux-gnu', '/usr/lib/aarch64-linux-gnu', '/lib', '/usr/lib']` for Linux on ARM64, etc.